### PR TITLE
pyca-cryptography: initial integration

### PR DESCRIPTION
**Pull request by arthurscchan**
initial fuzzer setup.
waiting for maintainer feedback.

### CODE REVIEW

File: projects/cryptography/symenc_fuzz.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+   token = key.encrypt(fdp.ConsumeBytes(32))
+   text = key.decrypt(token)
+
+   password = fdp.ConsumeBytes(8)
+   salt = fdp.ConsumeBytes(16)
+   kdf = PBKDF2HMAC(
+      algorithm=hashes.SHA256(),
+      length=32,
+      salt=salt,
+      iterations=390000,
```
Comment (alex) [inline projects/cryptography/symenc_fuzz.py:37]: Using a high iteration will make this slow, and doesn't improve the coverage here. At a minimum I'd set `iterations=1`.

File: projects/cryptography/symenc_fuzz.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+   )
+   key = Fernet(base64.urlsafe_b64encode(kdf.derive(password)))
+   token = key.encrypt(fdp.ConsumeBytes(32))
+   text = key.decrypt(token)
```
Comment (alex) [inline projects/cryptography/symenc_fuzz.py:41]: You could add an assert that this round-trips correctly.

File: projects/cryptography/project.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+fuzzing_engines:
+- libfuzzer
+homepage: https://github.com/pyca/cryptography
+language: python
+main_repo: https://github.com/pyca/cryptography
+sanitizers:
+- address
+primary_contact: "alex.gaynor@gmail.com"
```
Comment (DavidKorczynski) [inline projects/cryptography/project.yaml:8]: @alex FYI you're set as the primary contact -- is that okay?

Comment (alex) [reply projects/cryptography/project.yaml:8]: That's fine. @reaperhulk should also be added as a contact.

**Review by oliverchang — APPROVED** [review]

**Pull request merged**